src/lib/lumi.ts

```diff
@@ -4706,6 +4873,87 @@ export const toZigbee = {
             }
         },
     } satisfies Tz.Converter,
+    lumi_curtain_automatic_calibration_ZNCLDJ01LM: {
+        key: ['automatic_calibration'],
+        convertSet: async (entity, key, value, meta) => {
+            const NS = 'zhc:lumi:curtain';
```

Comment:
Please define this in the top of the file `const NS = 'zhc:lumi'`

Reply:
Should I use zhc:lumi instead or define zhc:lumi:curtain as another constant? I don't know what NS stands for.